tests/storage.spec.js

```diff
@@ -207,6 +207,51 @@ describe('Electron JSON Storage', function() {
 
   });
 
+  describe('.keys()', function() {
+
+    it('should return an empty array if no keys', function() {
+      const promise = storage.keys();
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).